### ISSUE TITLE
types for setActiveChannel

### ISSUE THREAD

**Issue by alexlopezit** [GetStream/stream-chat-react#261]
Hi there,

I'm wondering if the type definitions for [setActiveChannel](https://github.com/GetStream/stream-chat-react/blob/afe9a4279d1117fae3f4253264dd5c7bab61b21e/types/index.d.ts#L14) probably needs to be updated

The current interface definition is:
`setActiveChannel?(channel: Client.Channel, **event: React.SyntheticEvent**): void;`

But I think the second parameter should be optional and a different type?

**watchers?: SeamlessImmutable.Immutable<{ [user_id: string]: Client.User }>**

**Comment by DamonOehlman** [comment]
Another type definition issue in the recent releases:

```
node_modules/stream-chat-react/types/index.d.ts:872:30 - error TS2707: Generic type 'FC' requires between 0 and 1 type arguments.

872 export const LoadMoreButton: React.FC<LoadMoreButtonProps, any>;
```

Thought about creating a separate issue, but I think it's probably easiest if I add it here as I expect they will be cleaned up at the same time.

**Comment by jaapbakker88** [comment]
@DamonOehlman @alexlopezit these issues should be fixed in v1.0.0. Remember we started using hooks so minimum react version is now v16.8.x

**Comment by alexlopezit** [comment]
Thanks @jaapbakker88
